client/src/app/pages/filter/filter.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="general">
+    <h1>Filtrar</h1>
+
+    <div class="control">
+        <h2>Edad</h2>
+        <ul>
+            <li><input type="radio" name="age"> &lt; 1</li>
```

Comment:
You shold have used mat-select for this, maybe remake this using ion selects after the migration